datasets/multidoc2dial/README.md

```diff
@@ -0,0 +1,465 @@
+  - `doc_html_raw`: the document content with HTML markups and without span annotations.
+  - `spans`: key-value pairs of all spans in the document, with `id_sp` as key. Each span includes the following,
+    - `id_sp`: the id of a  span as noted by `text_id` in  `doc_html_ts`;
+    - `start_sp`/  `end_sp`: the start/end position of the text span in `doc_text`;
+    - `text_sp`: the text content of the span.
+    - `id_sec`: the id of the (sub)section (e.g. `<p>`) or title (`<h2>`) that contains the span.
+    - `start_sec` / `end_sec`: the start/end position of the (sub)section in `doc_text`.
+    - `text_sec`: the text of the (sub)section.
+    - `title`: the title of the (sub)section.
+    - `parent_titles`: the parent titles of the `title`.
+
+- **multidoc2dial_dial_train.json** and **multidoc2dial_dial_validation.json**  contain the training and dev split of dialogue data that are indexed by key `domain` . Please note: **For test split, we only include a dummy file in this version.**
+
+  Each dialogue instance includes the following,
+
+  - `dial_id`: the ID of a dialogue;
+  - `turns`: a list of dialogue turns. Each turn includes,
+    - `turn_id`: the time order of the turn;
+    - `role`: either "agent" or "user";READ
+    - `da`: dialogue act;
+    - `references`: a list of spans with `id_sp` ,  `label` and `doc_id`. `references` is empty if a turn is for indicating previous user query not answerable or irrelevant to the document. **Note** that labels "*precondition*"/"*solution*" are fuzzy annotations that indicate whether a span is for describing a conditional context or a solution.
+    - `utterance`: the human-generated utterance based on the dialogue scene.
+
+### Data Splits
+
+- Training & dev split for dialogue domain
+- Training split only for document domain
+
+## Dataset Creation
+
+### Curation Rationale
+
+[More Information Needed]
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+[More Information Needed]
+
+#### Who are the source language producers?
+
+[More Information Needed]
+
+### Annotations
+
+#### Annotation process
+
+[More Information Needed]
+
+#### Who are the annotators?
+
+[More Information Needed]
+
+### Personal and Sensitive Information
+
+[More Information Needed]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[More Information Needed]
+
+### Discussion of Biases
+
+[More Information Needed]
+
+### Other Known Limitations
+
+[More Information Needed]
+
+## Additional Information
+
+### Dataset Curators
+
+Song Feng, Siva Sankalp Patel, Hui Wan, Sachindra Joshi
+
+### Licensing Information
+
+Creative Commons Attribution 3.0 Unported
+
+### Citation Information
+
+@inproceedings{feng2021multidoc2dial,
+    title={MultiDoc2Dial: Modeling Dialogues Grounded in Multiple Documents},
+    author={Feng, Song and Patel, Siva Sankalp and Wan, Hui and Joshi, Sachindra},
+    booktitle={EMNLP},
+    year={2021}
+}
```

Comment:
add backticks for formatting
````suggestion
```bibtex
@inproceedings{feng2021multidoc2dial,
    title={MultiDoc2Dial: Modeling Dialogues Grounded in Multiple Documents},
    author={Feng, Song and Patel, Siva Sankalp and Wan, Hui and Joshi, Sachindra},
    booktitle={EMNLP},
    year={2021}
}
```
````